backend/backend/is_bankrupt.py

```diff
@@ -0,0 +1,48 @@
+"""Module for providing the functionality to
+   detirmine if a player is declared bankrupt
+"""
+
+import json
+import sys
+import cgitb
+from backend.player import Player
+from backend.game import get_games
+from backend.properties import Property, get_properties
+
+cgitb.enable()
+
+
+def is_bankrupt(source=sys.stdin, output=sys.stdout):
+    """Player is checked if they are bankrupt or not"""
+
+    output.write('Content-Type: application/json\n\n')
+    request = json.load(source)
+    player_id = request['player_id']
+
+    with Player(player_id) as player:
+        if player.balance < 0:
+            player_remove(player_id)
+
+
+def player_remove(player_id):
+    """Player is removed from game with their status updated
+       and appropriate changes commence (turn order, properties...)
+    """
+    games = get_games()
+    with Player(player_id) as player:
+        for game in games:
```

Comment:
We should write a function that gives the game(s) a player is in, so that we don’t have to do this iteration here. Would you be up for doing that as part of this pull request?

Reply:
@undergroundquizscene Are you suggesting the same type of function here as we were discussing in my pull request earlier? I think it would be very useful, in fact, I'm going to write one in my "tax" branch now. If someone else has it written already, then we can negotiate which one to keep at a later time.

Reply:
Thanks for pointing that out! I meant to come back to this, but forgot.

It’s up to you! Go for it if you want, but feel free to focus on other stuff if you feel that would be more useful, I’d say :)

Reply:
(Because there may be other things in general that are higher priorities – I’m uncertain how important this is)

Reply:
It's a good suggestion since it does seem to come up alot! I'll keep that part of the code the same if @arhh is writing up a function for getting the game the player is in. I could write one after I finishing making changes to this branch :)